Lib/argparse.py

```diff
@@ -571,16 +571,16 @@ def _format_action_invocation(self, action):
             #    -s, --long
             if action.nargs == 0:
                 parts.extend(action.option_strings)
+                return ', '.join(parts)
```

Comment:
`parts` is empty before this. So:
```suggestion
                return ', '.join(action.option_strings)
```